src/libraries/ExcelConverter/ExcelParser.cs

```diff
@@ -0,0 +1,420 @@
+﻿// <autogenerated>
+// Use autogenerated to suppress stylecop warnings 
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.Contracts;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Text.RegularExpressions;
+using DocumentFormat.OpenXml.InkML;
+using DocumentFormat.OpenXml.Packaging;
+using DocumentFormat.OpenXml.Spreadsheet;
+using DocumentFormat.OpenXml.Wordprocessing;
+using Microsoft.PowerFx;
+using Microsoft.PowerFx.Core;
+using Microsoft.PowerFx.Syntax;
+using Microsoft.PowerFx.Types;
+using Microsoft.PowerFx.Preview;
+using Microsoft.PowerFx.Interpreter;
+using Newtonsoft.Json;
+using DocumentFormat.OpenXml.Office2010.PowerPoint;
+using DocumentFormat.OpenXml.Drawing.Charts;
+
+namespace ExcelConverter
+{
+    public class ExcelParser
+    {
+        [JsonObject(ItemNullValueHandling = NullValueHandling.Ignore)]
+        public class ParsedExcelData
+        {
+            public List<ParsedCell> Cells;
+            public List<ParsedDefinedNames> DefinedNames;
+            public List<ParsedTable> Tables;
+        }
+
+        public class ParsedTable
+        {
+            public string Name;
+            public string Range;
+            public List<ParsedTableColumn> Columns;
+        }
+
+        [JsonObject(ItemNullValueHandling = NullValueHandling.Ignore)]
+        public class ParsedTableColumn
+        {
+            public string Name;
+            public string Formula;
+        }
+
+        [JsonObject(ItemNullValueHandling = NullValueHandling.Ignore)]
+        public class ParsedDefinedNames
+        {
+            public string Name;
+            public string Value;
+        }
+
+        [JsonObject(ItemNullValueHandling = NullValueHandling.Ignore)]
+        public sealed class ParsedCell
+        {
+            public string SheetName;
+            public string CellId;
+            public string Formula;
+            public string Value;
+        }
+
+     
+            public static ParsedExcelData ParseSpreadsheet(String excelFilePath, bool outputFile = true)
+            {
+                TestCellId();
+                TestSharedFormulaResolve();
```

Comment:
I think you can remove these from your code - this looks like something that was in the original parser just to test things, but it's not really needed for parsing the workbook

`TestCellId();
TestSharedFormulaResolve();`